all/experiments/parallel_env_experiment.py

```diff
@@ -138,5 +140,8 @@ def _fps(self, i):
         end_time = timer()
         return (self._frame - self._episode_start_frames[i]) / (end_time - self._episode_start_times[i])
 
-    def _make_writer(self, logdir, agent_name, env_name, write_loss):
-        return ExperimentWriter(self, agent_name, env_name, loss=write_loss, logdir=logdir)
+    def _make_writer(self, logdir, agent_name, env_name, write_loss, writer):
+        if writer == "comet":
+            return CometWriter(self, agent_name, env_name, loss=write_loss, logdir=logdir)
+        else:
```

Comment:
It is preferred not to have this `else` because the `if` block contains a `return`.